templates/monitoring/telemetry_missing_columns_v1/view.sql

```diff
@@ -0,0 +1,32 @@
+CREATE
+OR REPLACE VIEW `moz-fx-data-shared-prod.monitoring.telemetry_missing_columns_v1` AS
+SELECT
+  TIMESTAMP_TRUNC(submission_timestamp, DAY) AS day,
+  'telemetry' AS document_namespace,
+  REGEXP_EXTRACT(_TABLE_SUFFIX, r "^(.*)_v.*") AS document_type,
+  REGEXP_EXTRACT(_TABLE_SUFFIX, r "^.*_v(.*)$") AS document_version,
+  path,
+  COUNT(*) AS path_count
+FROM
+  `moz-fx-data-shared-prod.telemetry_stable.*`,
+  UNNEST(
+    udf_js.json_extract_missing_cols(
+      additional_properties,
+      ARRAY ["histogram_type"],
+      ARRAY [
```

Comment:
i think it's inconsistent with the use of other arrays in this repo to prefix arrays with `ARRAY` when it's not being used to specify the schema of the array's elements.

Reply:
I'm low-key in favor of prepending the type in all cases, it's pretty standard for SQL.

Reply:
that's fine, but we should use one or the other, rather than slowly introduce inconsistency.

Reply:
I'm going to use the version without the ARRAY prefix, it seems like the more natural way to write this.